tests/main/interfaces-browser-support/task.yaml

```diff
@@ -0,0 +1,138 @@
+summary: check that the browser-support interface works
+
+environment:
+    ALLOW_SANDBOX/allow: true
+    ALLOW_SANDBOX/disallow: false
+    OWNED_FILES: "/var/tmp/etilqs_test"
+    READABLE_FILES: "/run/udev/data/+platform:test
+    /etc/opt/chrome/test"
+    READABLE_WITH_SANDBOX_FILES: "/run/udev/data/c1:1-test
+    /run/udev/data/c10:1-test
+    /run/udev/data/c13:1-test
+    /run/udev/data/c180:1-test
+    /run/udev/data/c4:1-test
+    /run/udev/data/c5:1-test
+    /run/udev/data/c7:1-test
+    /run/udev/data/+hid:test
+    /run/udev/data/+input:input1-test
+    /run/udev/data/c29:1-test
+    /run/udev/data/+backlight:test
+    /run/udev/data/+leds:test
+    /run/udev/data/c116:1-test
+    /run/udev/data/+sound:card1-test
+    /run/udev/data/c108:1-test
+    /run/udev/data/c189:1-test
+    /run/udev/data/c89:1-test
+    /run/udev/data/c81:1-test
+    /run/udev/data/+acpi:test
+    /run/udev/data/+hwmon:hwmon1-test
+    /run/udev/data/+i2c:test"
+
+prepare: |
+    echo "Given a snap declaring a plug on browser-support with allow-sandbox set to $ALLOW_SANDBOX is installed"
+    cp -ar "$TESTSLIB/snaps/browser-support-consumer" .
+    sed "s/@ALLOW_SANDBOX@/$ALLOW_SANDBOX/" browser-support-consumer/meta/snap.yaml.in > browser-support-consumer/meta/snap.yaml
+    snapbuild browser-support-consumer browser-support-consumer
+    snap install --dangerous browser-support-consumer/*.snap
+    rm -rf browser-support-consumer
+
+restore: |
+    rm -f *.error /var/tmp/test
+    for file in $OWNED_FILES $READABLE_FILES $READABLE_WITH_SANDBOX_FILES; do
+        rm -f $file
+    done
+
+    for dir in $(cat created_dirs); do
+        rm -rf $dir
+    done
+    rm -f created_dirs
+
+execute: |
+    . "$TESTSLIB/dirs.sh"
+
+    CONNECTED_PATTERN=":browser-support +browser-support-consumer"
+    DISCONNECTED_PATTERN="^\- +browser-support-consumer:browser-support"
+
+    echo "Then the plug is connected by default"
+    snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    echo "And the snap is able to access tmp"
+    echo "test" | tee /var/tmp/test
+    su -l -c "$SNAPMOUNTDIR/bin/browser-support-consumer.cmd ls /var/tmp/" test | MATCH test
```

Comment:
If possible I usually try to run commands in snap tests as the least privileged user, so that root powers don't mask potential problems. Maybe it's useless or doesn't make sense, please let me know.